example/lib/test_scenarios/peripheral_test_operations.dart

```diff
@@ -0,0 +1,269 @@
+part of test_scenarios;
+
+class PeripheralTestOperations {
+  final Peripheral peripheral;
+  final Logger log;
+  final Logger logError;
+  StreamSubscription monitoringStreamSubscription;
+  final BleManager bleManager;
+
+  PeripheralTestOperations(
+      this.bleManager, this.peripheral, this.log, this.logError);
+
+  Future<void> connect() async {
+    log("Connecting to ${peripheral.name}");
+    await peripheral.connect();
+    log("Connected!");
+    return peripheral;
```

Comment:
What is the purpose of returning peripheral here since this method should return Future<'void'>?